pkg/core/event.go

```diff
@@ -73,27 +71,33 @@ type EventStore interface {
 
 // Event represents an event instance.
 type Event struct {
-	gorm.Model
```

Comment:
Why `gorm.Model` is not needed?

Reply:
The content in gorm.Model is "ID", "CreatedAt", "UpdatedAt", "DeletedAt". I took out these 4 things directly, so that it is convenient to add the "json" tag.

Reply:
Got it.